k8s-artifacts/mg-enforcer/config-toml-configmap.yaml

```diff
@@ -0,0 +1,78 @@
+# --------------------------------------------------------------------
```

Comment:
Lets use the same toml file for adapter and enforcer

Reply:
Fixed in 6b6708a8472501213c00230342810f801d5df4cf